contracts/test/unit/pcv/PCVRouter.t.sol

```diff
@@ -0,0 +1,336 @@
+        core.grantLocker(address(entry));
+        core.grantLocker(address(pcvOracle));
+        core.grantLocker(address(depositToken1Liquid));
+        core.grantLocker(address(depositToken1Illiquid));
+        core.grantLocker(address(depositToken2Liquid));
+        core.grantLocker(address(depositToken2Illiquid));
+        core.grantLocker(address(pcvRouter));
+        core.grantPCVController(address(pcvRouter));
+        core.createRole(VoltRoles.PCV_MOVER, VoltRoles.GOVERNOR);
+        core.grantRole(VoltRoles.PCV_MOVER, address(this));
+        vm.stopPrank();
+
+        // setup deposits
+        token1.mint(address(depositToken1Liquid), 100e18);
+        token1.mint(address(depositToken1Illiquid), 100e18);
+        token2.mint(address(depositToken2Liquid), 100e18);
+        token2.mint(address(depositToken2Illiquid), 100e18);
+        entry.deposit(address(depositToken1Liquid));
+        entry.deposit(address(depositToken1Illiquid));
+        entry.deposit(address(depositToken2Liquid));
+        entry.deposit(address(depositToken2Illiquid));
+
+        // setup PCVOracle
+        address[] memory venues = new address[](4);
+        venues[0] = address(depositToken1Liquid);
+        venues[1] = address(depositToken1Illiquid);
+        venues[2] = address(depositToken2Liquid);
+        venues[3] = address(depositToken2Illiquid);
+        address[] memory oracles = new address[](4);
+        oracles[0] = address(oracle);
+        oracles[1] = address(oracle);
+        oracles[2] = address(oracle);
+        oracles[3] = address(oracle);
+        bool[] memory isLiquid = new bool[](4);
+        isLiquid[0] = true;
+        isLiquid[1] = false;
+        isLiquid[2] = true;
+        isLiquid[3] = false;
+        vm.prank(addresses.governorAddress);
+        pcvOracle.addVenues(venues, oracles, isLiquid);
+
+        // setup PCVRouter
+        vm.prank(addresses.governorAddress);
+        pcvRouter.setPCVOracle(address(pcvOracle));
+    }
+
+    function testSetup() public {
+        assertEq(address(pcvRouter.core()), address(core));
+        assertEq(pcvRouter.pcvOracle(), address(pcvOracle));
+    }
+
+    // -------------------------------------------------
+    // Happy Path
+    // -------------------------------------------------
+
+    function testMovePCVEvents() public {
+        // PCVMovement
+        vm.expectEmit(true, true, false, true, address(pcvRouter));
+        emit PCVMovement(
+            address(depositToken1Liquid),
+            address(depositToken1Illiquid),
+            50e18
+        );
+
+        pcvRouter.movePCV(
+            address(depositToken1Liquid),
+            address(depositToken1Illiquid),
+            true,
+            false,
+            50e18
+        );
+    }
+
+    function testMovePCVBalances() public {
+        assertEq(depositToken1Liquid.balance(), 100e18);
+        assertEq(depositToken1Illiquid.balance(), 100e18);
+
+        pcvRouter.movePCV(
```

Comment:
add vm.expectCall to deposit() with address depositToken1Illiquid and vm.expectCall to withdraw(...) on depositToken1Liquid